.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Test suite
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches: [ main ]
+
+jobs:
+  lint-unittests:
+    name: Lint and Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.6, 3.8, "3.10"]
```

Comment:
will 3.10 be interpreted to 3.1 (float) without quotation marks? , I prefer to align the formate between  3.6/3.6/3.10, that is "all of them have quotaion mark". But it is not a big problem.

Reply:
Yes, it will be interpreted as 3.1. I'll add all of them into quotation marks.